state/endpoint_bindings.go

```diff
@@ -550,8 +550,11 @@ func (b *Bindings) MapWithSpaceNames() (map[string]string, error) {
 	// Assume that b.bindings is always in space id format due to
 	// Bindings constructor.
 	for k, v := range b.bindingsMap {
-		if v == network.DefaultSpaceId || v == network.DefaultSpaceName {
-			retVal[k] = network.DefaultSpaceName
+
+		// TODO (hml) 2019-10-29
+		// investigate if v == "" is still necessary
+		if v == network.AlphaSpaceId || v == network.AlphaSpaceName || v == "" {
```

Comment:
I think this should *only* be a check for "" whereupon we return "".

Reply:
done